components/select/off-click.ts

```diff
@@ -0,0 +1,26 @@
+import {Directive, Host, HostListener, Input} from 'angular2/core';
```

Comment:
New Directive which will fire an event when a click occurs outside of the element